webapp/src/main/java/com/box/l10n/mojito/rest/textunit/TextUnitWS.java

```diff
@@ -335,6 +339,12 @@ public TMTextUnitIntegrityCheckResult checkTMTextUnit(
           e);
       result.setCheckResult(false);
       result.setFailureDetail(e.getMessage());
+
+      // Handle Integrity Exceptions for Slack warnings if enabled
+      if (integrityCheckNotifier != null) {
+        integrityCheckNotifier.handleIntegrityException(
+            e, textUnitCheckBody.getTmTextUnitId(), textUnitCheckBody.getContent());
```

Comment:
Similar to above comment, is it possible to trigger the notify logic from within the integrity checker?